docs/sophia.md

```diff
@@ -565,11 +569,10 @@ string`, `String.sha3(s)` and `Crypto.sha3(s)` will give different results on AE
 ### Authorization interface
 
 When a Generalized account is authorized, the authorization function needs
```

Comment:
Maybe I missed this one, but nevertheless, Auth.tx isn't in iris, so neither should its documentation.

Reply:
this file shall be deleted after the merge, it's just the big template for creating the separate files.